x-pack/plugins/security_solution/public/timelines/components/side_panel/new_user_detail/hooks.test.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook } from '@testing-library/react-hooks';
+import type { InstalledIntegration } from '../../../../../common/detection_engine/fleet_integrations';
+import { TestProviders } from '../../../../common/mock';
+import { MANAGED_USER_PACKAGE_NAME } from './constants';
+import { useManagedUser } from './hooks';
+
+const makeInstalledIntegration = (
+  pkgName = 'testPkg',
+  isEnabled = false
+): InstalledIntegration => ({
+  package_name: pkgName,
+  package_title: '',
+  package_version: '',
+  integration_name: '',
+  integration_title: '',
+  is_enabled: isEnabled,
+});
+
+const mockUseInstalledIntegrations = jest.fn().mockReturnValue({
+  data: [],
+});
+
+jest.mock(
+  '../../../../detections/components/rules/related_integrations/use_installed_integrations',
+  () => ({
+    useInstalledIntegrations: () => mockUseInstalledIntegrations(),
+  })
+);
+
```

Comment:
`nit` : Does `useObservedUserItems` also need tests?

Reply:
It only calls some hooks and returns their values. So, the only test I can think of is to assert that the hooks are called with the correct parameters. It doesn't sound worthy of a test to me because it is very declarative, and we already have TS checking the type and order of parameters.